packages/nuxt/src/app/components/nuxt-island.ts

```diff
@@ -35,7 +35,7 @@ async function loadComponents (source = '/', paths: NuxtIslandResponse['componen
   for (const component in paths) {
     if (!(components!.has(component))) {
       promises.push((async () => {
-        const chunkSource = join(source, paths[component].chunk)
+        const chunkSource = join(source, appBaseURL, paths[component].chunk)
```

Comment:
shouldn't this instead be that `loadComponents` is _passed_ `appBaseURL` rather than it joins it to the source?

I imagine a remote source might not have the same baseURL

Reply:
oh indeed !

Reply:
Should it look maybe like this 
```js
let chunkSource
if(isRemoteSource){
      chunkSource = join(source, paths[component].chunk)
 }else{
     chunkSource = join(source, appBaseURL, paths[component].chunk)
}     
```
I assume that props.source would contain the remote url like `https://my.cdn.com/nuxt`
The signature would be the same and it makes sense that we dont need the baseUrl when source starts with "http"

Reply:
probably something like 
making source undefined by default in the signature then
```ts
if (!source) {
  source = appBaseURL ?? '/'
}
```

?